PSPrintSDK/OLFrameSelectionViewController.m

```diff
@@ -36,26 +36,26 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     vc.assets = self.assets;
     if ([segue.identifier isEqualToString:@"Selected2x2FrameStyleSegue"]) {
         for (OLProduct *product in products){
-            if (product.templateType == kOLTemplateTypeFrame2x2){
+            if (product.productTemplate.quantityPerSheet == 4){
```

Comment:
Ah makes sense. We should compare with the Frame Class as well. Now we would only miss the case where we could have more than one Frame template with the same quantity to fulfill order. Is that something we anticipate?